src/main/scala/no/ndla/searchapi/integration/SearchApiClient.scala

```diff
@@ -26,21 +26,21 @@ trait SearchApiClient {
     val searchPath: String
     def search(searchParams: SearchParams): Future[Try[ApiSearchResults]]
 
-    def get[T](path: String, params: (String, Any)*)(implicit mf: Manifest[T]): Try[T] =
```

Comment:
I serien "Rune forsøker å forstå scala". Hva gjør den stjerna du har fjernet her?

Reply:
Det er Scala sin syntax for "variabelt antall parametre. I praksis betyr det at man kan sende hvor mange tupler av typen `(String, Any)` til `get`-funksjonen. `params` får typen `List[(String, Any)]`.